internal/rootcoord/root_coord.go

```diff
@@ -1119,6 +1122,13 @@ func (c *Core) Init() error {
 			c.impTaskKv,
 			c.CallImportService,
 		)
+		// init data
+		encryptedRootPassword, _ := crypto.PasswordEncrypt(util.DefaultRootPassword)
+		initError = c.MetaTable.AddCredential(&internalpb.CredentialInfo{Username: util.UserRoot, EncryptedPassword: encryptedRootPassword})
+		if initError != nil {
+			return
+		}
+		log.Debug("RootCoord init user root done")
```

Comment:
the log of init function is a little bit messed up.

If initErr not nil, there should only one log about the error in WARN
if initErr not nil, there should be only one info log to show init successfully

Reply:
another pr for this